tox.ini

```diff
@@ -5,9 +5,8 @@ skipsdist = True
 [testenv]
 basepython = python3
 deps =
-    isort: isort
-    flake8: flake8
-            pyflakes
+    -r{toxinidir}/requirements.txt
```

Comment:
Do we need these requirements for linting? If we can skip installing all of Django and its requirements (and our 3rd party apps etc.) we can probably save quite some time.